### PR TITLE
Sabrina/2.0.0 beta.0

### PR DESCRIPTION
**Pull request by sabrina-li**
version bump

### CODE REVIEW

**Review by ScottLNorvell — APPROVED** [review]
lgtm

**Review by patrick-fs — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 2.0.0-beta.0
+
+- Convert to TypeScript
+- Change default export to named export `FullStory` 
+- Update `@fullstory/snippet` dependency to use version `2.0.0-beta.2`
+
```
Comment (patrick-fs) [inline CHANGELOG.md:8]: The big one: adopts the V2 browser API - https://developer.fullstory.com/browser/v2/getting-started/

Comment (patrick-fs) [inline CHANGELOG.md:8]:
```suggestion
- Implements the V2 beta browser API: https://developer.fullstory.com/browser/v2/getting-started/
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 2.0.0-beta.0
+
+- Convert to TypeScript
+- Change default export to named export `FullStory` 
+- Update `@fullstory/snippet` dependency to use version `2.0.0-beta.2`
+- Implements the v2 beta Browser API: https://developer.fullstory.com/browser/v2/getting-started/
```
Comment (patrick-fs) [inline CHANGELOG.md:8]:
```suggestion
- Implement the v2 beta Browser API: https://developer.fullstory.com/browser/v2/getting-started/
```

**Pull request merged**